lib/ruby_drills/array/select_drill.rb

```diff
@@ -0,0 +1,31 @@
+class SelectDrill < Drill
+
+  def setup
+    @values = [0, 1, 1, 2, 3, 5, 8, 13, 21, 34]
+    @hints = ["Are you being selective enough?",
+              "This method is called filter in other languages.",
+              "http://ruby-doc.org//core-2.2.0/Enumerable.html#method-i-select"]
+  end
+
+  def show
+    puts %{
```

Comment:
You'd better use a squiggly heredoc `<<~` introduced in Ruby 2.3 https://infinum.co/the-capsized-eight/articles/multiline-strings-ruby-2-3-0-the-squiggly-heredoc.